### ISSUE TITLE
Failed to create session

### ISSUE THREAD

**Issue by nagarajhubli** [nasa/NTL-Asteroid-Data-Hunter#1]
I am getting error saying "Failed to create session" when I start hunting after selecting the data files to hunt. Please find the error screenshot in the URL below

https://drive.google.com/file/d/0B7eJ4gOB47zqb0lJQXFfTzRXY00/view

I am using Windows 8.1 Pro, with 8GB of RAM

**Comment by rashid-sial** [comment]
@nagarajhubli, is the CMD window still open when you attempt the search? Also, are you using the sample files provided, or other .FITS files?

**Comment by nagarajhubli** [comment]
Yes, the CMD window was still open and the application was running, and I was using the Test Images as provided here.. http://www.topcoder.com/asteroids/asteroiddatahunter/

**Comment by nagarajhubli** [comment]
I found the cause of the issue, after looking at the error in the CMD window, which said...

Caused by:
org.h2.jdbc.JdbcSQLException: The database is read only; SQL statement:|insert into detection_session (id, progress) values (null, ?) [90097-176]

I realised that I had to right click start.bat and run as administrator.

**Comment by rashid-sial** [comment]
Thanks for the update @nagarajhubli!

**Comment by astrochoupe** [comment]
I had the same issue and resolved it by running as administrator. I also have an issue during the installation, fixed it the same way.

On Windows, if you install the app in "Program files" directory (that's the most logical directory to install it), you will accounter these problems because this directory is not writable by a common user!

The installation guide should mention it. Or the data should be written in the user's directory (c:\users...).

**Comment by rashid-sial** [comment]
Thanks @astrochoupe, I'm reopening this ticket until we resolve both of the issues mentioned above.
